SqlAssignments.sql

```diff
@@ -0,0 +1,1846 @@
+         FROM Orders GROUP BY ODATE,SNUM;
++------+------------+------------+
+| snum | odate      | Max_Amount |
++------+------------+------------+
+| 1001 | 1990-03-10 |     767.19 |
+| 1002 | 1990-03-10 |    5160.45 |
+| 1004 | 1990-03-10 |     1900.1 |
+| 1007 | 1990-03-10 |    1098.16 |
+| 1002 | 1990-04-10 |      75.75 |
+| 1003 | 1990-04-10 |    1713.23 |
+| 1001 | 1990-05-10 |       4723 |
+| 1001 | 1990-06-10 |    9891.88 |
+| 1002 | 1990-06-10 |    1309.95 |
++------+------------+------------+
+9 rows in set (0.00 sec)
+
+
+
+
+
+**************************************************************************************************************************************
+
+
+
+
+
+Que 4 ->  Arrange the Order table by descending customer number. 
+
+Query ->  SELECT onum, amt, odate, cnum,snum  
+          FROM Orders ORDER  BY cnum DESC;
++------+---------+------------+------+------+
+| onum | amt     | odate      | cnum | snum |
++------+---------+------------+------+------+
+| 3001 |   18.69 | 1990-03-10 | 2008 | 1007 |
+| 3006 | 1098.16 | 1990-03-10 | 2008 | 1007 |
+| 3002 |  1900.1 | 1990-03-10 | 2007 | 1004 |
+| 3008 |    4723 | 1990-05-10 | 2006 | 1001 |
+| 3011 | 9891.88 | 1990-06-10 | 2006 | 1001 |
+| 3007 |   75.75 | 1990-04-10 | 2004 | 1002 |
+| 3010 | 1309.95 | 1990-06-10 | 2004 | 1002 |
+| 3005 | 5160.45 | 1990-03-10 | 2003 | 1002 |
+| 3009 | 1713.23 | 1990-04-10 | 2002 | 1003 |
+| 3003 |  767.19 | 1990-03-10 | 2001 | 1001 |
++------+---------+------------+------+------+
+10 rows in set (0.00 sec)
+
+
+
+
+
+**************************************************************************************************************************************
+
+
+
+
+
+
+Que 5 ->  Find which salespeople currently have orders in the order table. 
+
+Query ->   SELECT DISTINCT sname  
+           FROM salespeople 
+           INNER  JOIN Orders ON salespeople.snum = Orders.snum;
++---------+
+| sname   |
++---------+
+| Peel    |
+| Serres  |
+| AxelRod |
+| Motika  |
+| Rifkin  |
++---------+
+5 rows in set (0.00 sec)
+
+
+
+
+
+
+**************************************************************************************************************************************
+
+
+
+Que 6 -> List names of all customers matched with the salespeople serving them. 
+
+Query -> SELECT cust.cnum,cname  
+      -> FROM Customers cust 
+      -> INNER JOIN salespeople sale ON cust.snum=sale.snum;
++------+----------+
+| cnum | cname    |
++------+----------+
+| 2001 | hoffman  |
+| 2002 | Giovanni |
+| 2003 | Liu      |
+| 2004 | Grass    |
+| 2006 | Clemens  |
+| 2007 | Pereira  |
+| 2008 | Cisneros |
++------+----------+
+7 rows in set (0.00 sec)
+
+
+
+
+**************************************************************************************************************************************
+
+
+
+
+
+Que 7 ->  Find the names and numbers of all salespeople who have more than one customer. 
+
+Query ->   
+         -> SELECT S.snum, S.sname, COUNT(C.snum) AS count  
```

Comment:
This is a wrong query or we can say incomplete.

Reply:
Okay, Sir, I'll correct.